RELEASE.md

```diff
@@ -0,0 +1,26 @@
+# Releases
+This page describes the release process and the currently planned schedule for upcoming releases as well as the respective release shepherd. Release shepherds are chosen on a voluntary basis.
+
+## Release Schedule
+
+Release cadence of first pre-releases being cut is 6 weeks.
+
+| release series | date (year-month-day) | release shepherd              |
+|----------------|-----------------------|-------------------------------|
+| v0.26          | 2023-05-31            | Josh Abreu (Github: @gotjosh) |
+| v0.27          | 2023-07-15            | Josh Abreu (Github: @gotjosh) |
+
+If you are interested in volunteering please create a pull request against the [prometheus/alertmanager](https://github.com/prometheus/alertmanager) repository and propose yourself for the release of your choice.
+
+## Release shepherd responsibilities
+
+The release shepherd is responsible for the entire release series of a minor release. 
```

Comment:
Do you mean to say that the release shepherd should actively look after all pull requests during the 6 weeks period?